test/steps/maven.go

```diff
@@ -28,13 +28,22 @@ func registerMavenSteps(s *godog.Suite, data *Data) {
 // Build local service
 func (data *Data) localServiceBuiltByMaven(serviceName string) error {
 	serviceRepositoryPath := data.KogitoExamplesLocation + "/" + serviceName
-	_, err := framework.CreateCommand("mvn", "clean", "package", "-DskipTests").InDirectory(serviceRepositoryPath).WithLoggerContext(data.Namespace).Execute()
+	output, err := framework.CreateMavenCommand(serviceRepositoryPath).
+		SkipTests().
+		WithLoggerContext(data.Namespace).
```

Comment:
Did you set the dot at the end or the IDE formatter did it automatically?
I used to see this in Java starting each line with the dot. 
Ignore this if this is a golang thing that was done by the formatter.

Reply:
I think that the compiler has a problem if the dot is at the beginning of the line and not at the end of the previous. let me check

Reply:
confirmed, I get a `expected statement, found '.'` syntax error